pkg/util/env_resolver.go

```diff
@@ -52,3 +56,27 @@ func ResolveOsEnvDuration(envName string) (*time.Duration, error) {
 
 	return nil, nil
 }
+
+func GetClusterObjectNamespace() (string, error) {
```

Comment:
could you please add description here to a comment?

Reply:
Sure, added.